doc/releases/migration-guide-3.7.rst

```diff
@@ -379,11 +379,12 @@ Bluetooth Audio
 ===============
 
 * :kconfig:option:`CONFIG_BT_ASCS`, :kconfig:option:`CONFIG_BT_PERIPHERAL` and
-  :kconfig:option:`CONFIG_BT_ISO_PERIPHERAL` are not longer `select`ed automatically when
+  :kconfig:option:`CONFIG_BT_ISO_PERIPHERAL` are not longer ``select``ed automatically when
```

Comment:
"The inline literal requires whitespace or punctuation around (as all inline-level elements)." so maybe just drop it altogether?

Reply:
Oh, is that a requirement? From whom? 

Never seen a formatter that did not support writing things like "`select`ed".

Reply:
It just happens to be ReSTructured text syntax https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup
I guess "` ``select`` `-ed" would work?

Reply:
Oh well, modified the text to avoid the issue. 

I guess 
```
`:code:`select`ed 
```

may also have worked